include/internal/common.h

```diff
@@ -75,7 +75,7 @@ __owur static inline int ossl_assert_int(int expr, const char *exprstr,
  */
 # define OSSL_UNION_ALIGN       \
     double align;               \
-    ossl_uintmax_t align_int;   \
+    uintmax_t align_int;   \
```

Comment:
align backslash with previous lines
```suggestion
    uintmax_t align_int;        \
```